library/swift/test/RetryPolicyTests.swift

```diff
@@ -1,7 +1,40 @@
-import Envoy
+@testable import Envoy
 import XCTest
 
 final class RetryPolicyTests: XCTestCase {
+  // MARK: - Conversions
+
+  func testConvertingToHeadersWithPerRetryTimeoutIncludesAllHeaders() {
+    let policy = RetryPolicy(maxRetryCount: 123,
+                             retryOn: RetryRule.allCases,
+                             perRetryTimeoutMS: 9001)
+    let expectedHeaders = [
+      "x-envoy-max-retries": "123",
+      "x-envoy-retry-on": RetryRule.allCases
```

Comment:
maybe expand `allCases` here to literals

Reply:
Done